format-xls/src/main/java/io/cdap/plugin/format/xls/input/XlsInputFormatConfig.java

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.format.xls.input;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Strings;
+import io.cdap.cdap.api.annotation.Description;
+import io.cdap.cdap.api.annotation.Macro;
+import io.cdap.cdap.api.annotation.Name;
+import io.cdap.cdap.api.data.schema.Schema;
+import io.cdap.cdap.api.dataset.lib.KeyValue;
+import io.cdap.cdap.api.plugin.PluginPropertyField;
+import io.cdap.plugin.common.KeyValueListParser;
+import io.cdap.plugin.format.input.PathTrackingConfig;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import javax.annotation.Nullable;
+
+/**
+ * Common config for Excel related formats.
+ */
+public class XlsInputFormatConfig extends PathTrackingConfig {
+  public static final String SHEET_NUMBER = "Sheet Number";
+  private static final String NAME_OVERRIDE = "override";
+  private static final String NAME_SHEET = "sheet";
+  public static final String NAME_SHEET_VALUE = "sheetValue";
+  private static final String NAME_SKIP_HEADER = "skipHeader";
+  private static final String NAME_TERMINATE_IF_EMPTY_ROW = "terminateIfEmptyRow";
+
+  // properties
+  public static final String NAME_SAMPLE_SIZE = "sampleSize";
+
+  public static final String DESC_SKIP_HEADER =
+    "Whether to skip the first line of each file. The default value is false.";
+  public static final String DESC_SHEET = "Select the sheet by name or number. Default is 'Sheet Number'.";
+  public static final String DESC_SHEET_VALUE = "Specifies the value corresponding to 'sheet' input. " +
+    "Can be either sheet name or sheet no; for example: 'Sheet1' or '0' in case user selects 'Sheet Name' or " +
+    "'Sheet Number' as 'sheet' input respectively. Sheet number starts with 0.";
+  public static final String DESC_TERMINATE_ROW = "Specify whether processing needs to be terminated in case an" +
```

Comment:
After reading the implementation, it looks like this is used to stop reading when an empty row is found. Let's change the wording here, I thought 'terminated' meant it would throw an error.

`Whether to stop reading after encountering the first empty row. Defaults to false.`

Reply:
> I assume it's being carried over from the existing excel source

Yes
 
- Updated the description for `DESC_TERMINATE_ROW`